draft-ietf-rats-architecture.md

```diff
@@ -734,10 +734,49 @@ or interval when changes in operational status, health, and so forth occur.
 
 ## Endorsements
 
-An Endorsement is a secure statement that some entity (e.g., a manufacturer) vouches for the integrity of the
-device's signing capability.  For example, if the signing capability is in hardware, then
-an Endorsement might be a manufacturer certificate that signs a public key whose corresponding
-private key is only known inside the device's hardware.  Thus, when Evidence and such an Endorsement
+Endorsements are essential input to the Verifier that allows them to
```

Comment:
singular / plural agreement

Reply:
I disagree with this phrasing.   Endorsements are essential for scalability.  If a Verifier owner has some means to establish trust in each Attester's hardware individually then it need not use an endorsement.  But typically that's not scalable.   Thus I prefer the original wording.

Reply:
The architecture needs to include the Verifier knows to trust the Attester, how it gets the key material on which the trust is based.  Since the only inputs are Endorsements and Policy, Endorsement seems the better line up.

Reply:
I prefer the old text.